exercises/more_string_manipulation/challenge_082.py

```diff
@@ -0,0 +1,37 @@
+# MORE STRING MANIPULATION CHAPTER
+
+# Challenge 082
+"""
+Show the user a line of text from your favourite poem and ask for a starting
+and ending point. Display the characters between those two points.
+"""
+
+
+def trim_characters(poem, start, end):
+    length_poem = len(poem)
+    if length_poem < start:
+        raise IndexError("list index out of range")
+
+    if isinstance(start, int) and isinstance(end, int):
+        if start > 0 and end >= start:
+            user_line_choose = poem[start:end]
+            return user_line_choose
+        else:
+            raise TypeError("Input indexes are out of range")
```

Comment:
Why is this a `TypeError` when the first bound-check is an `IndexError`? This seems inconsistent

Reply:
amended exceptions raised. Thanks for noticing!